lib/harmony/harmonydb/sql/20231113.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE harmony_task_history ADD COLUMN completed_by_host_and_port varchar(300) NOT NULL;
```

Comment:
tbh given that we're not rolled out anywhere yet, I'd say that we can afford to modify the main schema files, we can just drop the lotus schema between test runs;

(Not a super strong opinion)